session_analyzer.py

```diff
@@ -67,7 +66,7 @@ def analyze_paths(self, paths):
         for path in paths:
             if path['response_status'] is not 200:
                 errors += 1
-            if path['path'] in dorks:
+            if path['path'] in DorksManager.dorks:
```

Comment:
I'm not sure that is really good decision, but I really don't want to import the server here